docs/src/create-apps/source-operations.md

```diff
@@ -76,57 +101,55 @@ Run the following command:
 platform source-operation:run {{< variable "OPERATION_NAME" >}}
 ```
 
-Replace {{< variable "OPERATION_NAME" >}} with the name of your operation, such as `update-file` in the [example above](#1-define-a-source-operation).
+Replace {{< variable "OPERATION_NAME" >}} with the name of your operation, such as `update-file` in the [example above](#define-a-source-operation).
 
 {{< /codetabs >}}
 
-After running a source operation,
-apply the changes to your local development environment by running `git pull`.
-
-## How source operations work
-
-When you trigger a source operation, the following happens in order:
-
-1. The current environment HEAD commit is checked out in Git.
-   It doesn't have any remotes or tags defined in the project.
-   It only has the current environment branch.
-2. Sequentially, for each app that has an operation with this name in its configuration,
-   the operation command is run in the app container.
-   The container isn't part of the environment's runtime cluster
-   and doesn't require that the environment is running.
-
-   The environment has all of the variables normally available during the build phase.
-   These may be optionally overridden by the variables specified when the operation is run.
-3. If any new commits were created, they're pushed to the repository and the normal build process is triggered.
-
-   If multiple apps in a single project both result in a new commit,
-   there are two distinct commits in the Git history but only a single new build process.
+After running a source operation, 
+to apply the changes to your local development environment run the `git pull` command.
 
 ## Use variables in your source operations
 
-You can add variables to the environment of the source operation.
+You can add [variables](../development/variables/_index.md) to the environment of the source operation.
 
-They're interpreted the same way as any other [variable](../development/variables/_index.md)
-already set in your project.
-That means you need an `env:` prefix to expose them as a Unix environment variable.
+Use the `env:` prefix to expose each of those variables as a Unix environment variable.
+In this way, they're referenced by the source operation
+and interpreted the same way as any other variable set in your project.
 
-They can then be referenced by the source operation like any other variable.
+For example, you might want to have a `FILE` variable available with the value `example.txt`
+to pass to a source operation similar to the following:
 
-Say you wanted to have a `FILE` variable available with the value `example.txt`.
-Run the operation with the variable:
+```yaml {location=".platform.app.yaml"}
+source:
+    operations:
+        update-file:
```

Comment:
To match the command provided after this, this name needs to be `update`, or updated to `update-file` in the CLI command that comes next.

Reply:
If I had to choose, I would update [this line](https://github.com/platformsh/platformsh-docs/blob/e4b79172555d22f95b2f74809088d81f3505a517/docs/src/create-apps/source-operations.md?plain=1#L160) to `update-file`, since we're showing a variation on the previous example

Reply:
👍

Reply:
Agreed, done!